qiskit/circuit/library/__init__.py

```diff
@@ -345,6 +345,7 @@
    QuantumVolume
    PhaseEstimation
    GroverOperator
+   BitFlipOracle
```

Comment:
perhaps this should be done in another PR, like @Cryoris's https://github.com/Qiskit/qiskit/pull/13520, but I think that the 3 oracles (grover, phase and bitflip) deserve their own section in the circuit library docs.